components/cu_transform/examples/basic_usage.rs

```diff
@@ -0,0 +1,53 @@
+use cu29::clock::CuDuration;
+use cu_spatial_payloads::Transform3D;
+use cu_transform::{StampedTransform, TransformTree};
+
+fn main() {
+    let mut tree = TransformTree::<f32>::new();
+
+    let world_to_robot = StampedTransform {
+        transform: {
+            let mut t = Transform3D::default();
+            t.mat[0][3] = 1.0; // X translation
+            t.mat[1][3] = 2.0; // Y translation
+            t
+        },
+        stamp: CuDuration(1000),
```

Comment:
would a StampedTransform be simply a CuMsg<Transform>? (with have all the timestamps for the TOV in there or this is another semantic?)